app/helpers/utils.rb

```diff
@@ -5,7 +5,7 @@ module Utils
   # @return array of valid emails
   def self.parse_mails(mails)
     mails.split(/[\s,\,]/).select do |email|
-      email.present? && URI::MailTo::EMAIL_REGEXP.match(email)
+      email.present? && /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.match(email)
```

Comment:
You can use Devise::email_regexp instead

Reply:
I did it but the bot says i shouldn't use "::".